app/views/index/global.phtml

```diff
@@ -54,7 +57,7 @@
 		}
 	}
 
-	if ($unreadArticles < 1) {
+	if ($unreadArticles < 1 && $state_unread == true) {
```

Comment:
```suggestion
	if ($unreadArticles < 1 && $state_unread) {
```

Let's keep the boolean syntax simple

Reply:
yes, you are right. It was kept after "playing around"